README.md

```diff
@@ -193,6 +193,20 @@ This codebase follows [single-direction margin declarations](https://csswizardry
 
 Margins are generally preferred to padding for whitespace (due to [margin collapsing](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Box_Model/Mastering_margin_collapsing)), but the latter can be used when relevant.
 
+Accessibility
+------------
+
+### Screen Reader Observations
+
+#### Voiceover (Mac)**
```

Comment:
Maybe orientate this around the code rather than a particular reader, so this heading could be "`aria-label`  on `<a>` elements"

Reply:
This only applies to Safari with Voiceover so having the header defines it as an issue with this rather than someone scanning and thinking the problem is on all screen readers

Reply:
It's more about framing it in terms of what elements cause a problem.